### PR TITLE
Refactoring dynamo integration tests

### PR DESCRIPTION
**Pull request by thongdk8**
This PR does refactoring the integration tests for the DynamoDB adapter and adds checking empty metadata before being put. Please take a look. Thank you!

### CODE REVIEW

File: core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java [hunk 1]
```diff
@@ -214,16 +214,20 @@ public void addTableMetadata(String namespace, String table, TableMetadata metad
...
-        CLUSTERING_KEY,
-        AttributeValue.builder()
-            .l(
-                metadata.getClusteringKeyNames().stream()
-                    .map(pKey -> AttributeValue.builder().s(pKey).build())
-                    .collect(Collectors.toList()))
-            .build());
-    itemValues.put(
-        SECONDARY_INDEX, AttributeValue.builder().ss(metadata.getSecondaryIndexNames()).build());
+    if (!metadata.getClusteringKeyNames().isEmpty()) {
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java:217]: Do we need this change in this PR?

**Review by brfrn169 — COMMENTED** [review]
Thank you! LGTM overall. Left one question. Please take a look!

File: core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java (hunk 1 above)
Comment (thongdk8) [reply core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java:217]: Yes, we need it. The empty field in the request of putting in dynamo will throw an exception so I fix it as well.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**